LICENSE_APACHE

```diff
@@ -0,0 +1,71 @@
+Apache License, Version 2.0
+
+Version 2.0, January 2004
+
+http://www.apache.org/licenses/
+
+TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+
+1. Definitions.
+
```

Comment:
Why is there 3 license files in this project ? Only 2 are required for this project
1. Base license file
2. Apache2.0 license file (for newly added code)